src/rpc/RPCHelpers.cpp

```diff
@@ -1652,60 +1649,58 @@ traverseTransactions(
     boost::json::array txns;
     auto [blobs, retCursor] = transactionFetcher(
         context.backend, limit, forward, cursor, context.yield);
-    auto serializationStart = std::chrono::system_clock::now();
-
-    if (retCursor)
-    {
-        boost::json::object cursorJson;
-        cursorJson[JS(ledger)] = retCursor->ledgerSequence;
-        cursorJson[JS(seq)] = retCursor->transactionIndex;
-        response[JS(marker)] = cursorJson;
-    }
-
-    for (auto const& txnPlusMeta : blobs)
-    {
-        if ((txnPlusMeta.ledgerSequence < minIndex && !forward) ||
-            (txnPlusMeta.ledgerSequence > maxIndex && forward))
+    auto timeDiff = util::timed([&, &retCursor = retCursor, &blobs = blobs]() {
```

Comment:
Doesn't `&` already cover `retCursor` and `blobs`?

Reply:
it should be use explicitly, otherwise the compiler will complain the retCursor is not used :(.

Reply:
Ha! in this case you can either
```cpp
[[maybe_unused]] auto [blobs, retCursor] = ...;
```
i can see how this can be undesired. so maybe we can do this as a compromise:
```cpp
util::timed([&, &retCursor, &blobs]() { ... });
```

Reply:
![Screenshot 2022-12-14 at 15 46 33](https://user-images.githubusercontent.com/120398799/208631714-51e87dff-b231-492a-9726-d21d6b9d6c64.png)